speechbrain/lobes/models/convolution.py

```diff
@@ -133,6 +138,8 @@ def __init__(
                 stride=stride if i == num_layers - 1 else 1,
                 dilation=dilation,
                 layer_name=f"conv_{i}",
+                bias=conv_bias,
+                padding=padding,
```

Comment:
I am not sure about the padding. Padding is managed by SpeechBrain, this is why we don't provide it. No ?

Reply:
This follows fairseq where the input to the feature extractor is not padded. I followed that because I think it makes sense to not bother doing SSL training with representations that are on the "edge".